lib/DDG/Spice/WordMap.pm

```diff
@@ -12,38 +12,38 @@ category 	"reference";
 topics 		"words_and_games", "everyday";
 code_url 	"https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/WordMap.pm";
 icon_url 	"/i/www.twinword.com.ico";
-attribution web 	=> ['http://twinword.com','Twinword Inc.'],
-            twitter => ['http://twitter.com/levelpump', 'Twinword Inc.'],
+attribution web 	=> ['https://www.twinword.com','Twinword Inc.'],
+            twitter => ['https://twitter.com/twinwordinc', 'Twinword Inc.'],
             email 	=> ['feedback@twinword.com','Twinword Inc.'];
 
-spice to => 'http://duckduckgo.twinword.com/api/v2/context/user/duckduckgo/?entry=$1&api_key={{ENV{DDG_SPICE_WORDMAP_APIKEY}}}';
+spice to => 'https://www.twinword.com/api/v2/context/user/duckduckgo/?entry=$1&api_key={{ENV{DDG_SPICE_WORDMAP_APIKEY}}}';
```

Comment:
If possible we prefer `http` over `https`. We're proxying the request for the client through an HTTPS connection with DuckDuckGo so using `http` is a little faster.